src/clj/ataru/hakija/hakija_application_service.clj

```diff
@@ -292,3 +294,29 @@
   [feedback]
   (log/info "Saving feedback" feedback)
   (application-store/add-application-feedback feedback))
+
+(defn- attachment-metadata->answer [{:keys [fieldType] :as answer}]
+  (cond-> answer
+          (= fieldType "attachment")
+          (update :value (fn [value]
+                           (if (and (vector? value)
+                                    (not (empty? value))
+                                    (every? vector? value))
+                             (map file-store/get-metadata value)
+                             (file-store/get-metadata value))))))
+
+(defn attachments-metadata->answers [application]
+  (update application :answers (partial map attachment-metadata->answer)))
+
+(defn get-latest-application-by-secret [secret tarjonta-service person-client]
+  (let [application (-> secret
+                             (application-store/get-latest-application-by-secret)
+                             (flag-uneditable-answers tarjonta-service)
+                             (attachments-metadata->answers))
+
+        person (when application
```

Comment:
Täs pitäis varmaan olla vielä `(some? (:person-oid application))` sen varalta, että henkilönluonti ei oo vielä menny läpi?

Reply:
:+1:

Reply:
Itseasiassa varmaan halutaan palauttaa hakemukselta noi tiedot jos person(oid) on nil, eli jääköön tuolleesti.